model/proto/api_v2/query.proto

```diff
@@ -0,0 +1,154 @@
+import "protoc-gen-swagger/options/annotations.proto";
+
+option go_package = "api_v2";
+option java_package = "io.jaegertracing.api_v2";
+
+// Enable gogoprotobuf extensions (https://github.com/gogo/protobuf/blob/master/extensions.md).
+// Enable custom Marshal method.
+option (gogoproto.marshaler_all) = true;
+// Enable custom Unmarshal method.
+option (gogoproto.unmarshaler_all) = true;
+// Enable custom Size method (Required by Marshal and Unmarshal).
+option (gogoproto.sizer_all) = true;
+// Enable registration with golang/protobuf for the grpc-gateway.
+option (gogoproto.goproto_registration) = true;
+
+option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
+  info: {
+    version: "1.0";
+  };
+  external_docs: {
+    url: "https://github.com/jaegertracing/jaeger";
+    description: "Jaeger API";
+  }
+  schemes: HTTP;
+  schemes: HTTPS;
+};
+
+message GetTraceRequest {
+  bytes trace_id = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.customtype) = "github.com/jaegertracing/jaeger/model.TraceID",
+    (gogoproto.customname) = "TraceId"
+  ];
+}
+
+message GetTraceResponseStream {
+  repeated jaeger.api_v2.Span spans = 1 [
+    (gogoproto.nullable) = false
+  ];
+}
+
+message ArchiveTraceRequest {
+  bytes trace_id = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.customtype) = "github.com/jaegertracing/jaeger/model.TraceID",
+    (gogoproto.customname) = "TraceId"
+  ];
+}
+
+message ArchiveTraceResponse {
+}
+
+message TraceQueryParameters {
+  string service_name = 1;
+  string operation_name = 2;
+  map<string, string> tags = 3;
+  google.protobuf.Timestamp start_time_min = 4 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false
+  ];
+  google.protobuf.Timestamp start_time_max = 5 [
+    (gogoproto.stdtime) = true,
+    (gogoproto.nullable) = false
+  ];
+  google.protobuf.Duration duration_min = 6 [
+    (gogoproto.stdduration) = true,
+    (gogoproto.nullable) = false
+  ];
+  google.protobuf.Duration duration_max = 7 [
+    (gogoproto.stdduration) = true,
+    (gogoproto.nullable) = false
+  ];
+  int32 num_traces = 8;
+}
+
+message FindTracesRequest {
+  TraceQueryParameters query = 1;
+}
+
+message FindTracesResponse {
+  repeated jaeger.api_v2.Span spans = 1  [
+    (gogoproto.nullable) = false
+  ];
+}
+
+message GetServicesRequest {}
+
+message GetServicesResponse {
+  repeated string services = 1;
+}
+
+message GetOperationsRequest {
+  string service = 1;
+}
+
+message GetOperationsResponse {
+  repeated string operations = 1;
+}
+
+service QueryService {
+    rpc GetTrace(GetTraceRequest) returns (GetTraceResponseStream) {
+        option (google.api.http) = {
+            get: "/traces/{trace_id}"
+        };
+    }
+
+    rpc ArchiveTrace(ArchiveTraceRequest) returns (ArchiveTraceResponse) {
+        option (google.api.http) = {
+            post: "/traces/{trace_id}"
+        };
+    }
+
+    rpc Search(FindTracesRequest) returns (FindTracesResponse) {
```

Comment:
`rpc FindTraces(FindTracesRequest) returns (stream SpansResponseChunk);`

https://github.com/jaegertracing/jaeger/pull/1323/files#diff-ac4433b4ef3a403411d1d1b5141eb9ccR126

Reply:
Made the changes. Thanks!